CMakeLists.txt

```diff
@@ -60,6 +62,35 @@ endif()
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+if(ENABLE_IPO)
+    include(CheckIPOSupported)
+    check_ipo_supported(RESULT result OUTPUT output LANGUAGES CXX)
+    if(result)
+        message(STATUS "IPO / LTO enabled")
+	set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)    
+    	set(NJOBS 2)
```

Comment:
I prefer `-lfto=auto`, it is more complicated for user to specify a job number in `ENABLE_IPO`.

Reply:
clang not working. i cannot fix it.

Reply:
> I prefer `-lfto=auto`, it is more complicated for user to specify a job number in `ENABLE_IPO`.

fixed